v2/blockstore/readwrite.go

```diff
@@ -295,11 +295,23 @@ func (b *ReadWrite) PutMany(blks []blocks.Block) error {
 	for _, bl := range blks {
 		c := bl.Cid()
 
-		// Check for IDENTITY CID. If IDENTITY, ignore and move to the next block.
-		if _, ok, err := isIdentity(c); err != nil {
-			return err
-		} else if ok {
-			continue
+		// If IncludeIdentityCIDs option is disabled then treat IDENTITY CIDs like IdStore.
+		if !b.opts.IncludeIdentityCIDs {
+			// Check for IDENTITY CID. If IDENTITY, ignore and move to the next block.
```

Comment:
I wonder if you really want to convey `IncludeIden...` vs what actually happens: `StoreIden...`